core/storage/storage_models_test.py

```diff
@@ -176,14 +181,16 @@ def test_get_models_which_should_be_exported(self):
                     base_models.EXPORT_POLICY.NOT_APPLICABLE,
                     export_policy
                 )
-        # TODO(#8523): This number should be reduced to zero. This number
-        # should not be changed by developers under any circumstance.
+        # TODO(#8523): This list should not be modified under any circumstance.
+        # The export_data functions for the models in this list will eventually
+        # be implemented, and the TO_BE_IMPLEMENTED value will be removed.
         # Contact @varun-tandon for more information.
         expected_unimplemented = {
             collection_models.CollectionRightsSnapshotContentModel,
             collection_models.CollectionRightsSnapshotMetadataModel,
             collection_models.CollectionSnapshotContentModel,
             collection_models.CollectionSnapshotMetadataModel,
+            collection_models.CollectionModel,
```

Comment:
Why is this added?

Reply:
https://github.com/oppia/oppia/pull/8210#discussion_r376107939

Reply:
See above comment for why this is incorrect.

Reply:
Done.